plugins/openshift-developer/skills/address-review-pr/scripts/check_authorized.py

```
@@ -0,0 +1,205 @@
+            f"repos/{owner}/{repo}/contents/OWNERS_ALIASES",
+        ])
+        if HAS_YAML:
+            data = yaml.safe_load(content)
+            if data and "aliases" in data:
+                aliases = data["aliases"]
+        else:
+            aliases = _parse_simple_yaml_aliases(content)
+
+        for members in aliases.values():
+            authorized.update(members)
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)